pkgs/by-name/go/golem/package.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  stdenv,
+  darwin,
+  protobuf,
+  redis,
+  fontconfig,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "golem";
+  version = "1.0.16";
+
+  src = fetchFromGitHub {
+    owner = "golemcloud";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-XfSy6MGkEuEAhrF9LQJvHM786fFrBhYNJ4cIX+1eCvo=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    protobuf
+  ];
+  buildInputs =
+    [
+      fontconfig
+      openssl.dev
+    ]
+    ++ lib.optionals stdenv.isDarwin [
+      #darwin.apple_sdk.frameworks.SystemConfiguration
```

Comment:
from debugging?

Reply:
I have no mac, so I left this here for reference and someone to try it out as I assume it won't build without some platform specific dependencies.